ledger/participant-state/kvutils/tools/BUILD.bazel

```diff
@@ -125,3 +126,31 @@ da_scala_benchmark_jmh(
         "@maven//:org_scalaz_scalaz_core_2_12",
     ],
 )
+
+da_scala_benchmark_jmh(
+    name = "benchmark-codec",
+    srcs = glob(["codec-benchmark/src/benchmark/scala/**/*.scala"]),
+    data = [
+        "//ledger/participant-state/kvutils:reference-ledger-export.out",
+    ] if not is_windows else [],
+    max_heap_size = "4g",
+    scalacopts = lf_scalacopts,
+    tags = [
+        "manual",
+    ],
+    deps = [
+        "//bazel_tools/runfiles:scala_runfiles",
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/archive:daml_lf_dev_archive_proto_java",
+        "//daml-lf/data",
+        "//daml-lf/engine",
+        "//daml-lf/interpreter",
+        "//daml-lf/language",
+        "//daml-lf/transaction",
+        "//daml-lf/transaction:transaction_proto_java",
+        "//daml-lf/transaction:value_proto_java",
+        "//ledger/participant-state/kvutils",
+        "@maven//:com_google_protobuf_protobuf_java",
+        "@maven//:org_scalaz_scalaz_core_2_12",
```

Comment:
Is scalaz used in this module?

Reply:
Good catch. No, and apparently Buildozer is disabled on `scala_benchmark_jmh` rule instances. Don't know why. Removing.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8272/commits/642cce9316172827f62a41ea6652f86d452a278a

Reply:
I'm an idiot, I removed the dependency from the wrong rule.

Reply:
Yes, it's needed. Reverting.

```
> bazebuildun //ledger/participant-state/kvutils/tools:benchmark-codec
INFO: Invocation ID: 1b743167-678b-4e7e-b09e-0e9a12ca65a3
INFO: Analyzed target //ledger/participant-state/kvutils/tools:benchmark-codec (1 packages loaded, 24 targets configured).
INFO: Found 1 target...
ERROR: /home/ste/Projects/digital-asset/daml/ledger/participant-state/kvutils/tools/BUILD.bazel:130:23: scala //ledger/participant-state/kvutils/tools:benchmark-codec_generator failed (Exit 1): scalac failed: error executing command 
  (cd /home/ste/.cache/bazel/_bazel_ste/74ca99b07c4389a5a21de1bd00106db4/execroot/com_github_digital_asset_daml && \
  exec env - \
  bazel-out/k8-opt/bin/external/io_bazel_rules_scala/src/java/io/bazel/rulesscala/scalac/scalac @bazel-out/k8-opt/bin/ledger/participant-state/kvutils/tools/benchmark-codec_generator_scalac_worker_input)
Execution platform: @io_tweag_rules_nixpkgs//nixpkgs/platforms:host
ledger/participant-state/kvutils/tools/codec-benchmark/src/benchmark/scala/com/daml/lf/DecodeTransactionBenchmark.scala:26: error: Symbol 'type scalaz.Equal' is missing from the classpath.
This symbol is required by 'method com.daml.lf.value.Value.VersionedValue.VersionedValue$u0020Equal$u0020instance'.
Make sure that type Equal is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
A full rebuild may help if 'Value.class' was compiled against an incompatible version of scalaz.
    encodedTransactions.map(assertDecode)
                            ^
one error found
one error found
Build failed
java.lang.RuntimeException: Build failed
	at io.bazel.rulesscala.scalac.ScalacWorker.compileScalaSources(ScalacWorker.java:259)
	at io.bazel.rulesscala.scalac.ScalacWorker.work(ScalacWorker.java:79)
	at io.bazel.rulesscala.worker.Worker.persistentWorkerMain(Worker.java:92)
	at io.bazel.rulesscala.worker.Worker.workerMain(Worker.java:46)
	at io.bazel.rulesscala.scalac.ScalacWorker.main(ScalacWorker.java:46)
Target //ledger/participant-state/kvutils/tools:benchmark-codec failed to build
INFO: Elapsed time: 1.517s, Critical Path: 1.03s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
```

Reply:
Hmm. Ok, so Bazel modules don't see transitive deps. Didn't know that